examples/SampleUtils.cpp

```diff
@@ -163,7 +265,11 @@ namespace utils {
     }
 
     ml::Graph Build(const ml::GraphBuilder& builder, const std::vector<NamedOperand>& outputs) {
+#if defined(WEBNN_ENABLE_WIRE)
```

Comment:
Should we move both paths (wire enabled or not) into `CreateCppNamedOperands`?

Reply:
Yes, the flag should be passed with command line args in the future.

Reply:
I mean move the conditional code into `CreateCppNamedOperands`, so sample could just call it without caring about wire or not.